### PR TITLE
fr: Complete the translation for the page "Running the Course".

### PR DESCRIPTION
**Pull request by AdrienBaudemont**

### CODE REVIEW

**Comment by mgeisler** [comment]
@sakex and @lb034582341, this PR is waiting for your review (check https://github.com/pulls/review-requested frequently to see the PRs you're assigned to).

**Review by lb034582341 — APPROVED** [review]
Looks good to me, except one nit. Merci !

File: po/fr.po [hunk 1]
```diff
@@ -1258,13 +1267,18 @@ msgid ""
...
-"répartir les quatre jours sur deux semaines. Les participants au cours ont "
-"dit qu'ils trouvent utile d'avoir une pause dans le cours, car cela les aide "
-"à traiter toutes les informations que nous leur donnons."
+"Trouvez une salle suffisamment grande pour que tous les participants "
+"puissent suivre en personne. Nous recommandons une classe de 15 à 20 "
+"personnes. Cela constitue un groupe suffisamment petit pour que les gens "
+"soient à l'aise pour poser des questions --- c'est aussi assez petit pour "
+"qu'un instructeur ait le temps de répondre aux questions. Assurez-vous que "
+"la salle dispose de _bureaux_ pour vous et pour les étudiants : vous devriez "
+"tous pouvoir vous asseoir et travailler avec vos ordinateurs portables. En "
```
[the remark this answers is not in the document]
Comment (mgeisler) [reply po/fr.po:1276]: Thanks @lb034582341 for the review! Let me introduce you to GitHub suggestions, it's the left-most button in the toolbar above:

```suggestion
"tous pouvoir vous assoir et travailler avec vos ordinateurs portables. En "
```

**Comment by mgeisler** [comment]
Thanks @lb034582341! I went ahead and resolved the merge conflict so the PR could be merged. It should be live on https://google.github.io/comprehensive-rust/fr/ in a few minutes.

**Comment by AdrienBaudemont** [comment]
Thank you everyone!

**Pull request merged**